coalib/testing/LocalBearTestHelper.py

```diff
@@ -25,6 +25,21 @@ def execute_bear(bear, *args, **kwargs):
                 console_output.append('The program yielded '
                                       'the following output:\n')
                 old_process_output = bear.process_output
+                old_create_arguments = bear.create_arguments
+
+                def new_create_arguments(filename,
+                                         file,
+                                         config_file,
+                                         *args,
+                                         **kwargs):
+                    arguments = old_create_arguments(filename,
+                                                     file,
+                                                     config_file,
+                                                     *args,
+                                                     **kwargs)
```

Comment:
optional: Align maybe parameters not on a single line each, but if there's still enough space in the line place them directly besides. See `new_process_output` for example.